templates/create-frontend-plugin/template.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: create-frontend-plugin
+  title: Create Frontend Plugin Template
+  description: Template for creating a fontend plugin skeleton
```

Comment:
Typo on `fontend`

Reply:
I also created the [backend template](https://github.com/redhat-developer/red-hat-developer-hub-software-templates/pull/6), where I took your structure for the template.yaml and techdocs. Now I wonder if we should create a folder for our templates? Thanks!

Reply:
@jesuino Fixed the typo, and added documentation for dynamic plugin enablement. PTAL